pkgs/development/libraries/sycl-info/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv
+, fetchFromGitHub
+, installShellFiles
+, cmake
+, ninja
+, ocl-icd
+, opencl-headers
+, lyra
+, nlohmann_json
+, ronn
+, doctest
+}:
+
+let
+  rev = "b47d498ee2d6b77ec21972de5882e8e12efecd6c";
+in
+stdenv.mkDerivation rec {
+  pname = "sycl-info";
+  version = builtins.substring 0 7 rev;
```

Comment:
when not selecting from a release, the paradigm is to do `unstable-YYYY-MM-DD` of the commit